AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/AnyModelIntegrationTests.swift

```diff
@@ -42,10 +42,9 @@ class AnyModelIntegrationTests: XCTestCase {
     }
 
     func testCreateAsAnyModel() throws {
-
         let originalPost = Post(title: "Post title",
                                 content: "Original post content as of \(Date())",
-                                createdAt: Date())
+            createdAt: .now())
```

Comment:
indentation

Reply:
the line above 
```
content: "Original post content as of \(Date())",
```
with the string interpolation is what's causing xcode to have some sort of indentation bug there

Reply:
fixed by creating some variables